frontend/src/assets/i18n/en.json

```diff
@@ -600,13 +605,15 @@
             "and": "and",
             "base-changes": "Common changes",
             "area-changes": "Area-specific changes",
-            "no-changes": "No changes"
+            "no-changes": "No changes",
+            "overflow-notice": "Change exceeding maximum"
         },
         "impact-table": {
             "ecoComponent": "Impact per nature value",
             "pressure": "Impact per pressure",
             "cumulative": "Cumulative",
-            "percentage-of-total": "% of Total"
+            "percentage-of-total": "% of Total",
+            "overflow": "Maximum exceeded, entire change misreported"
```

Comment:
It may be better to use the phrase 'incorrectly reported' than 'misreported' here. 'Misreported' is frequently used when people are intentionally mislead to believe something different than what is true. If that is what is meant then it should definitely stay the same. 

> "overflow": "Maximum exceeded, entire change incorrectly reported"

Reply:
Good catch, thanks! 🙏